README.md

```diff
@@ -305,6 +251,7 @@ def after_perform(*args)
 end
 ```
 
+(no need to do this for Rails, but a Rack app maybe)
```

Comment:
This part may be confusing. For sidekiq and no rails, the default background thread removes the need for custom sidekiq middleware. The only custom code needed should be Coverband.configure and Coverband.start. I am thinking we should probably even remove the need for this. Maybe:

```ruby
if defined?(Sidekiq) && !defined?(Rails)
  Sidekiq.configure_server do
    Coverband.configure
     Coverband.start
  end
end
```

While we are on the topic of configure and start, would it be possible to just call these methods on require of coverband.rb and remove this requirement altogether?

Reply:
Yeah so I was thinking for Ruby but not Rails Sidekiq apps... I think your idea of automatically calling configure and start even outside of the railtie is an interesting one. Let me run some tests on some plain ruby scripts and see if I have any Sinatra apps to test with, It seems like a good idea, but I would want to test it out a bit on some samples first.

Reply:
ok, take a look at the proof of concept of this idea...

added on latest commit to this branch... https://github.com/danmayer/coverband/commit/3240a161f8739150ed18473647284914fffaaf97

An example script which collects its own runtime data via this method
https://github.com/danmayer/coverband_demo/pull/2/files#diff-7ce9c6873a144d3d0eb1ca7010c03a86R1

The Coverband demo Rails app still works as well.

Reply:
Let me know if you see any issues with this, if not I will try to do a bit more testing and clean up docs @kbaum

Reply:
removed all these docs as we have autostart on require working now